### ISSUE TITLE
Symlinks at ftp://ftp.nodc.noaa.gov/pub/data.nodc/

### ISSUE THREAD

**Issue by JeremiahCurtis** [climate-mirror/datasets#338]
Not captured in the mirrors at issues #122, #158, or #163 

There are hordes of symlinks  not mirrored at ftp://ftp.datadefendr.com/ftp.nodc.noaa.gov/pub/data.nodc/ .....when reviewing ftp://ftp.nodc.noaa.gov/pub/data.nodc/

Merely going through the following folders:
		
ghrsst/		
glider/		
go.gif	
goc/		
gocd	
gtspp/		
iode/		
ioos/		
jason2/		
jason3/		

I found the following missing from the mirror at ftp://ftp.datadefendr.com/ftp.nodc.noaa.gov/pub/data.nodc/:

ftp://ftp.nodc.noaa.gov/pub/data.nodc/jason2/gdr
ftp://ftp.nodc.noaa.gov/pub/data.nodc/jason2/igdr
ftp://ftp.nodc.noaa.gov/pub/data.nodc/jason2/ogdr
ftp://ftp.nodc.noaa.gov/pub/data.nodc/ioos/sccoos/newport_pier
ftp://ftp.nodc.noaa.gov/pub/data.nodc/ioos/sccoos/santa_monica_pier
ftp://ftp.nodc.noaa.gov/pub/data.nodc/ioos/sccoos/scripps_pier
ftp://ftp.nodc.noaa.gov/pub/data.nodc/ioos/sccoos/stearns_wharf
several dozen symlinks buried in ftp://ftp.nodc.noaa.gov/pub/data.nodc/ioos/secoora/
ftp://ftp.nodc.noaa.gov/pub/data.nodc/gtspp/best/
ftp://ftp.nodc.noaa.gov/pub/data.nodc/gtspp/best_inv/
ftp://ftp.nodc.noaa.gov/pub/data.nodc/gtspp/best_nc/
ftp://ftp.nodc.noaa.gov/pub/data.nodc/gtspp/indian/
ftp://ftp.nodc.noaa.gov/pub/data.nodc/gtspp/pacific/
ftp://ftp.nodc.noaa.gov/pub/data.nodc/gtspp/realtime/
ftp://ftp.nodc.noaa.gov/pub/data.nodc/gtspp/tools/
ftp://ftp.nodc.noaa.gov/pub/data.nodc/gocd
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/002/20040120
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/002/20041223
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/004/20031002
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20020911
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20030208
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20030821
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20040624
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20040830
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20050608
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20061109
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20070910
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20080606
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20081106
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/005/20090829
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/008/20031002
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/012/20041223
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/012/20050207
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/012/20051116
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/012/20070831
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/014/20040924
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/014/20060424
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/014/20070315
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/014/20080214
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/014/20080222
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/015/20040924
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/016/20050406
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/016/20061112
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/016/20071113
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/016/20080607
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/016/20090605
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/030/20061109
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/030/20080117
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/030/20080915
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/101/20061112
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/101/20070609
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/101/20080401
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/101/20081108
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/102/20061112
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/102/20071113
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/103/20070218
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/103/20071113
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/103/20090223
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/104/20070901
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/104/20080214
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/119/20080915
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/120/20080830
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/144/20080608
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/144/20090614
ftp://ftp.nodc.noaa.gov/pub/data.nodc/glider/seaglider/uw/157/20141116

probably several hundred or even thousands of symlinks at ftp://ftp.nodc.noaa.gov/pub/data.nodc/ghrsst/

There are, without a doubt, other symlinks not captured in the mirror

Does anyone know how to approach these? I'm a neophyte at ftp synchronization (and a far cry from a tech wizard), and I don't want to overlap with the mirror mentioned above (in order to save time and storage for other efforts). If anyone could tackle the symlinks, it would help immensely. At the moment , I'm using almost my entire bandwidth to simultaneously grab the three massive 100+ GB files at ftp://newftp.epa.gov/RSEI/Version235_RY2015/Disaggregated_Microdata/......I had a few hiccups, but the ETA for the trio is late Sunday evening

**Comment by defendr** [comment]
Yep, that's a problem -- there doesn't seem to be a way to do it that I could find:
https://forum.filezilla-project.org/viewtopic.php?t=23515

**Comment by gabefair** [comment]
@defendr @JeremiahCurtis that answer is incorrect. I've seen it done before, but my google skills are failing me ATM.
I have asked a stack overflow question. Hopefully we can get some answers: http://stackoverflow.com/q/42602015/635160
